omniledger/service/api.go

```diff
@@ -70,3 +68,27 @@ func (c *Client) GetProof(r *onet.Roster, id skipchain.SkipBlockID, key []byte)
 	}
 	return reply, nil
 }
+
+// DefaultGenesisMsg creates the message that is used to for creating the
+// genesis darc and block.
+func DefaultGenesisMsg(v Version, r *onet.Roster) CreateGenesisBlock {
+	ids := make([]*darc.Identity, len(r.List))
+	for i := range ids {
+		ids[i] = darc.NewIdentityEd25519(r.List[i].Public)
```

Comment:
I don't think we want to have the nodes' public keys in the genesis block for the darc. The darc should be given by the client who requests a new skipchain. The conodes per se don't have any voting power in the omniledger setup.